### PR TITLE
Send worker logs to syslog as well

### PR DESCRIPTION
**Pull request by jpopelka**
The syslog part will be a [fluentd](https://docs.fluentd.org/input/syslog) container [running in the same pod](https://github.com/packit/deployment/pull/352) and listening on the specified TCP port.

https://github.com/packit/research/pull/152

### CODE REVIEW

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
recheck

File: files/install-deps-worker.yaml [hunk 1]
```diff
@@ -52,6 +52,7 @@
         name:
           - git+https://github.com/packit/sandcastle.git@{{ source_branch }}
           - sentry-sdk
+          - syslog-rfc5424-formatter
```
Comment (TomasTomecek) [inline files/install-deps-worker.yaml:55]: such an easy to remember name...

**Review by TomasTomecek — APPROVED** [review]

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
The tests fail because of a missing dependency (`syslog-rfc5424-formatter`) added in this PR.
They pass locally.

**Pull request merged**